app/models/package.rb

```diff
@@ -300,10 +302,18 @@ def delete_associated_packages_locations
     packages_locations.destroy_all
   end
 
-  def update_allow_web_publish_to_false
+  def unpublish
     update(allow_web_publish: false)
   end
 
+  def publish
+    update(allow_web_publish: true)
+  end
+
+  def published?
+    return allow_web_publish.present?
```

Comment:
I think we don't need to add return here? 🤔 💭

Reply:
indeed we don't, I'm just new to ruby so I type it without thinking about it haha

Reply:
updated 👍